integrations/azure-devops/azure_devops/http/http_client.py

```diff
@@ -0,0 +1,150 @@
+import httpx
+from httpx import Timeout, BasicAuth, Response
+from typing import Any, AsyncGenerator, Optional
+from port_ocean.utils import http_async_client
+from loguru import logger
+
+PAGE_SIZE = 50
+REQUEST_TIMEOUT_SECONDS = 30
+CONTINUATION_TOKEN_HEADER = "x-ms-continuationtoken"
+ALLOW_REDIRECTS = True
+
+
+class HTTPBaseClient:
+    def __init__(self, personal_access_token: str) -> None:
+        self._async_client = http_async_client
+        # Username isn't required in basic auth to Azure Devops
+        self._auth = BasicAuth("", personal_access_token)
+        self._timeout = Timeout(REQUEST_TIMEOUT_SECONDS)
+        self._follow_redirects = ALLOW_REDIRECTS
+
+    @staticmethod
+    def _parse_response_values(response: Response) -> list[dict[Any, Any]]:
+        return response.json()["value"]
```

Comment:
cant u just always access the ["value"]?

Reply:
Wanted to hide this ugly part and not force every method to know about this key, should I change this? 
Doesn't really make a difference

Reply:
if everyone access the value key then yes

Reply:
also it is a bit of an overkill and it will be much more readable without it IMO